Automattic/wp-calypso
# Add publish confirmation toggle to Writing Settings

This PR seeks to make it possible for the user to re-enable the pre-publish confirmation screen so that they can confirm every time they publish a post.

Publishing confirmation toggle is at Settings > Writing > Composing.

![screen shot 2017-06-30 at 7 15 09 pm](https://user-images.githubusercontent.com/1017839/27746685-7c10f79e-5dc8-11e7-9b0b-7c0bd30f1188.png)

This PR is part of a larger epic. The feature is behind a feature-flag and is intentionally incomplete. See p8F9tW-3F-p2.

To test:

- ENABLE_FEATURES=post-editor/delta-post-publish-flow make run
- Navigate to Settings --> Writing
- Verify that a toggle to show publish confirmation is present in the Composing section
- Change the setting and verify that it persists between page reloads
- Switch to a different site and verify that the setting is per site

To verify that analytics works:
- Open the console and enable debugging `localStorage.setItem( 'debug', '*' );`
- Filter for `analyt`
- Change the toggle and verify that there is an mc and a tracks event fired

To verify that the setting is not visible without the feature flag enabled:
- `make run`
- Navigate to Settings --> Writing
- Verify that a toggle to show publish confirmation is NOT present
